pages/join.html

```diff
@@ -7,6 +7,42 @@
 
 <h2>Join content goes here.</h2>
 
+<div class="starting-projects-container">
+    <div class='starting-projects-content'>
+        <h2>How do we start projects?</h2>
+        <p>Code for Chicago is focused on making projects shoulder-to-shoulder with our partners.</p>
+    </div>
+    <div class='starting-projects-grid'>
+        <div class="starting-projects-item item-1">
```

Comment:
Instead of adding these very specific `item-1` and `item-2` classes that won't be reused, can we perhaps try a [lobotomized owl](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/) approach?

e.g.
```
starting-projects-grid > *+* {
   margin-left: 32px;
}
```


So, all children that have an adjacent item right before it will have a margin on the left.

Reply:
I learned something new today! Very useful!